compress/src/main/java/org/hyperledger/besu/nativelib/compress/LibCompress.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Besu Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+package org.hyperledger.besu.nativelib.compress;
+
+import com.sun.jna.Native;
+
+/**
+ * Java interface to compress
+ */
+public class LibCompress {
+
+    @SuppressWarnings("WeakerAccess")
+    public static final boolean ENABLED;
+
+    static {
+        boolean enabled;
+        try {
+            Native.register(LibCompress.class, "compress_jni");
+
+            // TODO don't know how to manage resources well (in the java project sense of the term)
+            // -- "compressor_dict.bin" should match 
+            // the version used in Linea.
```

Comment:
```
 // TODO don't know how to manage resources well (in the java project sense of the term)
            // -- "compressor_dict.bin" should match 
            // the version used in Linea.

            if (!Init("compressor_dict.bin")) {
                throw new RuntimeException(Error());
            }
```

--> what I meant is the "compressor_dict.bin" file should be managed as a resource of the caller project and the proper path injected there, not sure how to do that cleanly :)

Reply:
I think it is possible to give the path to the file in the project which will use the library and put it into resources of the project that use it. Like that it can be deleted from here. No?